### PR TITLE
Update volume/testing/OWNERS file

### PR DESCRIPTION
**Pull request by xing-yang**
#### What type of PR is this?
Add one of the following kinds:
/kind documentation

#### Which issue(s) this PR fixes:
Fixes #

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```docs
```

### CODE REVIEW

File: pkg/volume/testing/OWNERS [hunk 1]
```diff
@@ -1,14 +1,10 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
+- sig-storage-approvers
```
Comment (msau42) [inline pkg/volume/testing/OWNERS:4]: Should we let this bubble up to pkg/volume/OWNERS?

Comment (xing-yang) [reply pkg/volume/testing/OWNERS:4]: Sure.

**Pull request closed without merging**